service/src/main/java/uk/gov/mca/beacons/api/legacybeacon/application/LegacyBeaconService.java

```diff
@@ -97,7 +97,11 @@ public List<LegacyBeacon> delete(
     legacyBeaconData.setUses(new ArrayList<LegacyUse>());
     legacyBeaconData.setEmergencyContact(new LegacyEmergencyContact());
     legacyBeaconData.setSecondaryOwners(new ArrayList<LegacySecondaryOwner>());
+
     legacyDataBeaconDetails.setNote(null);
+    legacyDataBeaconDetails.setLastModifiedDate(
+      OffsetDateTime.now().toString()
```

Comment:
All looks good! Just one question from me:

1. Are there tests around what the legacy beacon should look like now?
1.1 Does the toString() give the correct date format compared to prod legacy beacons?

Reply:
1. ✅ 
1.1 Comparing the two, it looks like production's last modified date is in this format '2021-09-28T17:43:24' while mine locally is '2023-02-07T17:25:38.090747Z' with the extra timestamp on the end. I'll see about getting rid of the extra bit tomorrow

Reply:
1.1 Ok got it now! Looks like production's using a LocalDateTime so I'll use that instead of OffsetDateTime

Reply:
Local now shows "2023-02-08T10:02:44" being returned from the /legacy-beacon/[id] endpoint for the beacon I've just deleted